server/controllers/project.controller.js

```diff
@@ -140,14 +140,21 @@ export function projectExists(projectId, callback) {
   ));
 }
 
-export function projectForUserExists(username, projectId, callback) {
+export function projectForUserExists(username, authenicatedUser, projectId, callback) {
+  console.log('Just got called');
```

Comment:
Also I'd remove this console.log, sometimes they get left in for testing but I try to remove all of them.